runtime-modules/council/src/lib.rs

```diff
@@ -198,18 +220,29 @@ pub trait Trait: system::Trait {
     /// Duration of idle period
     type IdlePeriodDuration: Get<Self::BlockNumber>;
 
+    /// The value elected members will be awarded each block of their reign.
+    type ElectedMemberRewardPerBlock: Get<Balance<Self>>;
+    /// Interval for automatic reward payments.
+    type ElectedMemberRewardPeriod: Get<Self::BlockNumber>;
+
     /// Checks that the user account is indeed associated with the member.
     fn is_council_member_account(
         membership_id: &Self::MembershipId,
         account_id: &<Self as system::Trait>::AccountId,
     ) -> bool;
+
+    /// Converts block number type to balance type.
+    /// NOTE: This function is needed because there is no way to enforce type constraint
+    ///       `Trait::BlockNumber: From<Balance<T>> + Into<Balance<T>>` needed by `get_current_reward()`.
+    ///       That's because cascade of `where` clauses leads to `decl_error` macro that doesn't support
+    ///       type constraint via `where`.
+    fn blocks_to_balance(block_number: &Self::BlockNumber) -> Balance<Self>;
```

Comment:
I don't think should be required, have a look here

Reply:
Removed; problem solved itself after saturated arithmetic was introduced.